index.html

```diff
@@ -4,83 +4,219 @@
-					<img class="logo-img" src="./images/logo.png">
-					<h1 class="logo pop">
-						Virtision
-					</h1>
-					<h4 class="motto">Easily create, share and enjoy immersive worlds together</h4>
+					<h1 class="logo pop">The XR Social Music Platform</h1>
+					<h2 class="motto">Easily create, share and enjoy immersive worlds together</h2>
 					<div class="subscribe">
 						<h5 class="info-text">
 							Join the waiting list for the beta. We keep you posted.
 						</h5>
 						<div>
-							<form class="form-inline validate" role="form" action="https://virtision.us1.list-manage.com/subscribe/post?u=9ab8c2f01a3f5aa677094fc1e&amp;id=4c5085e68c" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank" novalidate>
-								<div class="form-group">
-									<label class="sr-only" for="exampleInputEmail2">Email address</label>
-									<input type="email" name="EMAIL" class="form-control transparent" placeholder="Your email here...">
-									<div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_9ab8c2f01a3f5aa677094fc1e_4c5085e68c" tabindex="-1" value=""></div>
+							<form class="form-inline validate" role="form" action="https://virtision.us1.list-manage.com/subscribe/post?u=9ab8c2f01a3f5aa677094fc1e&amp;id=4c5085e68c" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form"
+								target="_blank" novalidate>
+								<div class="row">
+									<div class="form-group">
+										<input type="email" name="EMAIL" class="form-control transparent" placeholder="Your email here...">
+										<div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_9ab8c2f01a3f5aa677094fc1e_4c5085e68c" tabindex="-1" value=""></div>
+									</div>
+									<button type="submit" class="btn btn-first btn-email btn-fill">NOTIFY ME</button>
 								</div>
-								<button type="submit" class="btn btn-email btn-fill">Notify Me</button>
-
 							</form>
-							<div class="media">
-								<a href="#">
+
+							<div class="media mb-3">
+								<a href="https://www.linkedin.com/company/virtision-com">
 									<i class="fa fa-linkedin-square"></i>
 								</a>
 
 								<a href="#">
 									<i class="fa fa-facebook"></i>
 								</a>
 
-								<a href="#">
-									<i class="fa fa-envelope-o"></i>
+								<a href="mailto:team@virtision.com">
+									<i class="fa fa-envelope"></i>
 								</a>
 
 							</div>
 						</div>
 					</div>
 				</div>
+				<div class="col-lg-5 offset-lg-1 float-right mb-3">
+					<video playsinline="" muted="" class="rounded shadow-lg video" src="images/demo.mp4" loop autoplay></video>
+				</div>
+
+			</div>
+		</div>
+	</div>
+	<div class="main">
+		<div class="container value">
+			<div class="row justify-content-center">
+				<div class="col-lg-8 center">
+					<h1 class="pop"> For any creatives at any level to create and share their imagination in XR</h1>
```

Comment:
Grammatically correct, and I like what this sentence is saying, but I maybe we could make it sound even better? 

"Creatives, from novice to expert, can share their imagination in XR."
"Calling all creatives of all skill levels: share your imagination in XR."
"For creatives to share their imagination in XR, whether they're beginners or experts."

Also maybe creatives -> creators? IDK.

Reply:
yeah creators is better since it is one of the popular keywords for social media nowadays